app/src/main/java/com/codecademy/movieapp/FavoritesFragment.kt

```diff
@@ -0,0 +1,59 @@
+package com.codecademy.movieapp
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+
+// TODO: Rename parameter arguments, choose names that match
+// the fragment initialization parameters, e.g. ARG_ITEM_NUMBER
+private const val ARG_PARAM1 = "param1"
+private const val ARG_PARAM2 = "param2"
+
+/**
+ * A simple [Fragment] subclass.
+ * Use the [FavoritesFragment.newInstance] factory method to
+ * create an instance of this fragment.
+ */
+class FavoritesFragment : Fragment(R.layout.fragment_favorites) {
```

Comment:
My first time to see the layout id being passed, and it seems to not crash anything, but I don't think we need this.
```suggestion
class FavoritesFragment : Fragment() {
```